src/redux/books/books.js

```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 
 const ADD_BOOK = 'react-bookstore/books/ADD_BOOK';
 const DELETE_BOOK = 'react-bookstore/books/DELETE_BOOK';
-const BOOK_FAILURE = 'react-bookstore/books/BOOK_FAILURE';
+const BOOK_FAIL = 'react-bookstore/books/BOOK_FAIL';
 
```

Comment:
- [ ] As requested by the previous code reviewer in this [PR](https://github.com/Victoria129/book-store-app/pull/9). You should these books into your initial state and display them on the page. You should use the [createSlice()](https://redux-toolkit.js.org/api/createSlice) function which is a function that accepts an initial state, an object of reducer functions.
```js
const initialState = {
  books: [
    {
      item_id: "item1",
      title: "The Great Gatsby",
      author: "John Smith",
      category: "Fiction",
    },
    {
      item_id: "item2",
      title: "Anna Karenina",
      author: "Leo Tolstoy",
      category: "Fiction",
    },
    {
      item_id: "item3",
      title: "The Selfish Gene",
      author: "Richard Dawkins",
      category: "Nonfiction",
    },
  ],
}
```